lib/forwardanalyzer.cpp

```diff
@@ -524,7 +524,7 @@ struct ForwardTraversal {
                     if (!condTok->hasKnownIntValue() || inLoop) {
                         if (!analyzer->lowerToPossible())
                             return Break(Analyzer::Terminate::Bail);
-                    } else if (condTok->values().front().intvalue == inElse) {
+                    } else if (condTok->values().front().intvalue == static_cast<long long>(inElse)) {
```

Comment:
I don't see the danger here and would prefer to turn off the msvc warning.

Reply:
Is it ok to insert `#pragma warning`s? There are some more places where warnings could be suppressed